synd-contracts/src/staking/GasAggregator.sol

```diff
@@ -16,51 +16,115 @@ interface AppchainFactory {
     function getAppchainsAndContracts() external view returns (uint256[] memory chainIDs, address[] memory contracts);
 }
 
-/// @title GasAggregator
-/// @notice Aggregates gas usage data from appchains and pushes it to the staking appchain
+/**
+ * @title GasAggregator
+ * @notice Aggregates gas usage data from appchains and pushes it to the staking appchain
```

Comment:
```suggestion
 * @notice Aggregates gas usage data from appchains
```

This contract doesn't push to the staking appchain, it instead keeps the epoch data hash in storage so it can be used in proofs